### ISSUE TITLE
→ Accessibility: Components | Avoid ellipsis in component labels

### ISSUE THREAD

**Issue by aweell** [Telefonica/mistica-design#2152]
## Description

We have identified a possible failure of https://www.w3.org/WAI/WCAG22/Understanding/resize-text.html in some Mística components  that truncate their label when the available space is less than their width.

The failure: https://www.w3.org/WAI/WCAG22/Techniques/failures/F69

Components affected: 

[Example of the issue](https://mistica-web.vercel.app/playroom/#?code=N4Igxg9gJgpiBcIA8AlGBnADhAduglgG4wAyAhgJ4QCuALgHwA6OABC0lES%2BrRQDYwAvMGAsAtmQAeAdXxRaAC3gsATAAY1LAL5amrNuwDCC-Jha5DfCOiHAAFAEoWg%2Bi2A76AFRPoW%2BX2QsVjgA5kFkAEYwfCyKZLSxAE7UOGDxGEgA9MamegbsnmRhvJhCjCCYiRBiEOUsAJKQOMKNuADKtGSJAGL4fAJQusz5bN7%2BfgFBuGF8kdGxCvFJKWm0GMMGWYUheZueMJK0vdFQLDhkYmUg55d1s1F8guVjvuOBwTNzMXEJtMmp6XQdUyuzYSDa0RgYFoG3yNyuADNkvhaECQLCDPdok8QC8Jix3tNwg8Fks-itAeUMWwIJhaPhcOhhABtakGUSEMh8agwZTlCCJMihGDlAA0sQOtD5IAA8oLhXUtKK2WwOVyedKIkLtWKJYdpQAhbXnRXK-T5AC6WgxIIxSANdFouAACol8BJEhRzDhXRgmfYnC43B42Xi3lNQsT5j9lgC1kDzexMg7aE6fW6PRRQewU2mIU0oF0vS7En7hI5nK53ENE6MfPjCZGsd9Fr9-qt1omsrn2lDcIXPdn7Y7cCR8DgANbe33of0VoPVrz18MfKMtsntwHdkc4MeT3ZZTiEPRZNBYRlEUiUGgMZggUUgRQwS7oBDMkAAWQghH8nUSIAtB8AHc5EUV94GZAB2AA2AAOUUAEYAGY4KtIA)

Proposal for a solution:

https://www.figma.com/design/LsQMW0UYytjHsLvaxEnzGC/Ellipsis-avoidance-in-components?node-id=0-1&t=yTjIFk0f4E61t5AU-1

## Tasks

- [ ] Update specs
- [ ] Design documentation
- [ ] Add to mobile library
- [ ] Add to desktop library
- [ ] Add to UI Kit
- [ ] Android ticket
- [ ] iOS ticket
- [ ] Web ticket

**Comment by yceballost** [comment]
Should we link this issue with accessibility project?
